web/html/src/manager/content-management/shared/components/panels/sources/channels/group-channels.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React from 'react';
+import {Toggler} from "components/toggler";
+import {Highlight} from "components/data-handler";
+import ChildChannels from "./child-channels";
+import type {ChannelsTreeType} from "core/channels/api/use-channels-tree-api";
+import type {RequiredChannelsResultType} from "core/channels/api/use-mandatory-channels-api";
+import {getAllRecommentedIdsByBaseId} from "core/channels/utils/channels-state.utils";
+
+type PropsType = {
+  base: Object,
+  search: string,
+  childChannelsId: Array<number>,
+  selectedChannelsIdsInGroup: Array<number>,
+  selectedBaseChannelId: number,
+  isOpen: boolean,
+  setAllRecommentedChannels: Function,
+  onChannelToggle: Function,
+  onOpenGroup: Function,
+  channelsTree: ChannelsTreeType,
+  requiredChannelsResult: RequiredChannelsResultType,
+}
+
+const GroupChannels = (props: PropsType) => {
+  const nrOfSelectedChilds = props.selectedChannelsIdsInGroup.length;
+
+  const {recommendedIds, areRecommendedChildrenSelected} = getAllRecommentedIdsByBaseId(
+    props.base.id,
+    props.channelsTree,
+    props.selectedChannelsIdsInGroup
+  );
+  const toggleRecommended = () =>
+    areRecommendedChildrenSelected
+      ? props.setAllRecommentedChannels(false)
+      : props.setAllRecommentedChannels(true)
+
+  return (
+    <div className='row'>
+      <h4 style={{
+        backgroundColor: props.base.id === props.selectedBaseChannelId ? "lightBlue" : "",
```

Comment:
What about just using a simple `color: #a42800;` instead of highlighting it with a background? I think we don't need to attract the attention but just to differentiate it from others because it will be the target new base channel. About the color I would use the `#a42800` red which is the same of the `mandatory` label, because this new target base channel is *mandatory* as well in a way, you know.